packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1090,6 +1094,216 @@ describe("SharedTree", () => {
 				});
 			}
 		});
+
+		describe("can resubmit", () => {
+			const sb = new SchemaBuilder({ scope: "shared tree undo tests" });
```

Comment:
Looks like `SchemaBuilder` is deprecated.

Reply:
If you want to work at the flex-tree abstraction level, you can use SchemaBuilderBase, some examples in https://github.com/microsoft/FluidFramework/pull/19255 . If you instead want to work with simple trees, thats a whole different API surface.

I'd also recommend doing with a simpler schema unless you actually need 3 distinct node types, and trees 3 levels deep. Maybe you can just use the already existing stringSequenceRootSchema, which is much simpler (only one kind of node, and trees of depth 1)

Reply:
3 distinct node types? list, string, and...?